### DISCUSSION TITLE
Add missing pseudo elements

### DISCUSSION THREAD

**Discussion by CroModder** [vanilla-extract-css/vanilla-extract#1201] [General]
Hi all, is there a reason why some pseudo element selectors are not included in [this list](https://github.com/vanilla-extract-css/vanilla-extract/blob/1dbd56c8292af03e53aa3f4b288a40d88bc762cf/packages/css/src/simplePseudos.ts)?
At first I thought that e.g. `::-webkit-progress-bar` is not included in there because it's an non standard feature but then I found out that also other non standard pseudo elements are already included in that list.
I would be happy to open PR if you want to support other non standard pseudo elements if you decide for that.

**Comment by askoufis** [comment]
[`::-webkit-progress-bar` is present in that list](https://github.com/vanilla-extract-css/vanilla-extract/blob/1dbd56c8292af03e53aa3f4b288a40d88bc762cf/packages/css/src/simplePseudos.ts#L33), but ultimately it's a hard-coded list that hasn't been updated in a while, so I'm not surprised that there are some missing pseudo-selectors that have been added to CSS in recent years. A PR would be welcome 😃.

**Reply by CroModder** [reply]
I meant to say [meter-bar](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-meter-bar) is missing, but went on mdn and copied wrong selector 😄 
 Ok, great, will prepare a PR in upcoming days

**Reply by askoufis** [reply]
Addressed in https://github.com/vanilla-extract-css/vanilla-extract/pull/1207
